### ISSUE TITLE
onSlide vs onSlideEnd cb functions

### ISSUE THREAD

**Issue by nikhilzac95** [andreruffert/rangeslider.js#271]
@andreruffert can u please list out the differences between these two callback functions? I was trying them out and could not clearly find when onSlideEnd would get triggered. Also, please do list down the pros and cons (if any) of using each one of those cb functions.

**Comment by andreruffert** [comment]
### `onSlide`
Is triggered every time you move the handle.

__Equivalent to:__

```js
$('input[type="range"]').on('input', function() {
  //...
});
```

### `onSlideEnd`
Is triggered only if you move and __release__ the handle.

__Equivalent to:__

```js
$('input[type="range"]').on('change', function() {
  //...
});
```

I'd suggest you to listen either to `input` or `change` events. The callback functions (`onSlide`, `onSlideEnd`) are just implemented for some edge cases where you can easily access e.g. the position of the handle, or other variables defined for the running instance.

**Comment by nikhilzac95** [comment]
@andreruffert thanks a lot. actually onSlideEnd cb function is the one i was searching for. 👍
